python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -449,6 +460,7 @@ async def kiloarcListenerLoop(self):
                     self.detailed_state = WLSDetailedState.OFFLINE
                 previousState = currentState
             except Exception as e:
+                # if there's a problem, try to reconnect
                 self.log.debug(e)
                 self.log.debug("Connection Problem with ADAM/Kiloarc")
```

Comment:
Please consider replacing these three lines with the following (log.exception prints the exception automatically; you do not have to save it to a variable such as "e":
```
except Exception:
    self.log.exception("Connection problem with ADAM/Kiloarc")
```